### ISSUE TITLE
Create header section of the site

### ISSUE THREAD

**Issue by vishalrangras** [chiragrangras/author-reflection-bootstrap-site#3]
- This header section should be added to all the 4 pages of the site

**Comment by chiragrangras** [comment]
Should I also add bootstrap in the project for the header section?
Should I use CDN version or local version of bootstrap?

**Comment by vishalrangras** [comment]
Yes, please add bootstrap to the project. Don't use CDN version since it could slow the loading of page. Please use the local version. Please store the bootstrap files in the directory ./src/common/css and ./src/common/js
